### PR TITLE
Return information about mount_program (fuse-overlayfs)

### PR DESCRIPTION
**Pull request by rhatdan**
We want to get podman info to tell us about the version of
the mount program to help us diagnose issues users are having.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@giuseppe @mheon @vrothberg @baude @TomSweeneyRedHat @QiWang19 @ashley-cui @jwhonce PTAL

File: libpod/util.go [hunk 1]
```diff
@@ -148,3 +152,36 @@ func JSONDeepCopy(from, to interface{}) error {
...
+	}
+	return dpkgVersion(program)
+}
+
+func programVersion(mountProgram string) (string, error) {
+	output, err := utils.ExecCmd(mountProgram, "--version")
+	if err != nil {
+		return "", err
+	}
+	return strings.TrimSuffix(strings.Replace(output, "\n", ", ", 1), "\n"), nil
```
Comment (vrothberg) [inline libpod/util.go]: I like trimming the trailing newline but `strings.Replace(output, "\n", ", ", 1)` may not be applicable other programs (i.e., when there's only one line). Can we change it to:
```golang
return strings.TrimSuffix(output, "\n"), nil
```
?

Comment (vrothberg) [reply libpod/util.go]: That's still present. @rhatdan, any comments?

**Pull request merged**